tasks/src/test/java/org/example/aProblemAboutEquation/AProblemAboutEquation3Test.java

```diff
@@ -0,0 +1,78 @@
+package org.example.aProblemAboutEquation;
+
+import lombok.extern.java.Log;
+import org.junit.Before;
+import org.junit.jupiter.api.*;
+
+import java.util.List;
+
+@Log
+class AProblemAboutEquation3Test {
+    private final AProblemAboutEquation3 aProblemAboutEquation3 = new AProblemAboutEquation3();
+
+
+    @Before
```

Comment:
Nie uzywaj Before bo Before to jest adnotacje ktora jest przestarzala 
Uzywa sie albo BeforeAll ( i wtedy metoda musi byc statyczna) lub BeforeEach i wtedy moze byc normalna